apropos.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Since your page content is in French, change HTML lang="en" to "fr".

Reply:
Done